addons/rating/__manifest__.py

```diff
@@ -22,10 +22,23 @@
     'assets': {
         'web.assets_backend': [
             'rating/static/src/scss/rating_rating_views.scss',
+            'rating/static/src/components/thread_needaction_preview/thread_needaction_preview.scss',
+            'rating/static/src/components/thread_preview/thread_preview.scss',
```

Comment:
Better use glob patterns to match all the *.scss files below the `components` directory:
```suggestion
            'rating/static/src/components/*/*.scss',
```
It's less tedious as you can create new components without having to update the manifest. But most importantly, it allows us to add new files in stable